.kiro/specs/neovim-config-modernization/research.md

```
@@ -0,0 +1,112 @@
+# 調査・設計意思決定ログ: neovim-config-modernization
+
+---
+**目的**: 技術設計に必要な調査結果、アーキテクチャ検討、および設計判断の根拠を記録する。
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)